clang/test/CXX/drs/cwg17xx.cpp

```diff
@@ -203,6 +203,9 @@ namespace cwg1762 { // cwg1762: 14
   float operator ""E(const char *);
   // since-cxx11-error@-1 {{invalid suffix on literal; C++11 requires a space between literal and identifier}}
   // since-cxx11-warning@-2 {{user-defined literal suffixes not starting with '_' are reserved; no literal will invoke this operator}}
+  // since-cxx11-warning@-3 {{identifier 'E' preceded by whitespace in a literal operator declaration is deprecated}}
```

Comment:
Wow, that's one strange warning. Are you sure we can deploy this patch like this?

Reply:
We kinda already do ship this!

 But yeah, I put a fixme there because I don't have a good way of figuring out how to avoid it.  Is there a good way of detecting that we added the space with a fixit, so that we can skip the warning in that case?

Reply:
After discussion (and to save this thought): What about making the "user-defined literal suffixes not starting with '_' are reserved" be a default error, then making this 'new' diagnostic only happen if the thing starts with an underscore.

I think that fixes THIS case as well as makes the non-fixit-change case actually sane?

Reply:
I agree that this might be a way forward.